test/integration/kernel.spec.ts

```diff
@@ -7341,93 +7341,6 @@ describe('Kernel', () => {
 			);
 		});
 
-		it('should report back action requests', (done) => {
```

Comment:
Moving this to `jellyfish-queue` (new home of the `action-request` contract) with https://github.com/product-os/jellyfish-queue/pull/1109